src/python/platforms/android/settings.py

```diff
@@ -18,13 +18,25 @@
 import re
 
 from . import adb
+from base import utils
+from platforms.android import symbols_downloader
 from system import environment
 
 BUILD_FINGERPRINT_REGEX = re.compile(
     r'(?P<vendor>.+)\/(?P<target>.+)'
     r'\/(?P<flavor>.+)\/(?P<name_name>.+)'
     r'\/(?P<build_id>.+):(?P<type>.+)\/(?P<keys>.+)')
 
+PRODUCT_TO_KERNEL = {
```

Comment:
probably lets move this to existing file android/constants.py , so that it is more prominent and will be updated over time.